packages/fuel-indexer-utils/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+pub use fuel_indexer_macros as macros;
+pub use fuel_indexer_plugin::*;
+pub use fuel_indexer_schema;
+pub use fuels;
```

Comment:
If we're going to allow `fuels` with this level of scope (public), thoughts on removing it as a dependency from indexers? Cause with this ☝🏽 I can just `use fuel_indexer_utils::fuels::` as opposed to having to manage another external dependency

Reply:
I was having quite a bit of trouble before with being able to remove it, but I think I may have fixed the issue with my last push, so I'll try this again. We'll be able to get rid of version mismatches that way. 👌🏽

Reply:
It appears that `fuels` has to stay as part of the `Cargo.toml` as the SDK codegen itself depends on some attributes that I can't seem to include in this utils crate. 👎🏽

Reply:
Ok in that case we can remove it here